com.stansassets.scene-management/Runtime/Statis/AdditiveScenesLoader.cs

```diff
@@ -459,14 +459,7 @@ static void ProcessSceneUnLoad(string sceneName)
 
         static bool ValidateScene(string sceneName)
         {
-            // TODO let's reconsider the validation
-            /*
-            if (!Application.isEditor && BuildConfigurationSettings.Instance.HasValidConfiguration)
-            {
-                return BuildConfigurationSettings.Instance.Configuration.HasScene(sceneName);
-            }*/
-
-            return true;
+            return BuildConfigurationSettings.Instance.Configuration.HasScene(sceneName);
```

Comment:
Looks like `BuildConfigurationSettings.Instance.HasValidConfiguration` was there for a reason. What if we just installed the plugin and we don't have a configuration yet?